Fix various typos in the scenarios readme (#3771)
DataDog/system-tests: docs/scenarios/README.md

## How to identify a scenario?

> Every scenarios is identified by an unique identifier in capital letter, like `APPSEC_IP_BLOCKING_FULL_DENYLIST`. To specify a scenario, simply use its name after `run.sh`:

Reviewer: ```suggestion
Every scenario is identified by a unique identifier in capital letters, like `APPSEC_IP_BLOCKING_FULL_DENYLIST`. To specify a scenario, simply use its name after `run.sh`:
```

Author: Resolving since I got this change in.

### End to end scenarios

> Based on class `EndToEndScenario`, they spawn a "weblog" HTTP server designed to mimick customer applications with automatic instrumentation, a "test-agent" to mimick the Datadog Agent, and communication with the Datadog backend via a proxy. The `DEFAULT` scenario is the main scenario of system tests, and is in this family.

Reviewer: ```suggestion
Based on the class `EndToEndScenario`, they spawn a "weblog" HTTP server designed to mimic customer applications with automatic instrumentation, a "test-agent" to mimic the Datadog Agent, and communication with the Datadog backend via a proxy. The `DEFAULT` scenario is the main scenario of system tests, and is in this family.
```

End-To-End scenarios are good for testing real-world scenarios — they support the full lifecycle of a trace (hence the name, "End-To-End"). Use End-To-End scenarios to test tracing integrations, security products, profiling, dynamic instrumentation, and more. When in doubt, use end-to-end.

### Parametric scenario

Parametric scenarios are designed to validate tracer and span interfaces. They are more lightweight and support testing features with many input parameters. They should be used to test operations such as creating spans, setting tags, setting links, injecting/extracting http headers, getting tracer configurations, etc. You can find dedicated parametric instructions in the [parametric.md](https://github.com/DataDog/system-tests/blob/main/docs/scenarios/parametric.md).

### Auto-Inject/Onboarding scenarios

> Automatic library injection simplifies the APM onboarding experience for customers deploying Java, Node.js, .NET, Python and Ruby applications in VMs and docker environments. Datadog software installed on the machine will be intercept the startup of your application and it will inject the tracer library automatically. The Onboarding scenarios reproduce different environments and check the library injection is done correctly. More detailed documentation can be found [here](https://github.com/DataDog/system-tests/blob/main/docs/scenarios/onboarding.md).

Reviewer: ```suggestion
Automatic library injection simplifies the APM onboarding experience for customers deploying Java, Node.js, .NET, Python and Ruby applications in VMs and containerized environments. Datadog software installed on the machine will intercept the startup of your application and inject the tracer library automatically. The Onboarding scenarios reproduce different environments and check that the library injection is done correctly. More detailed documentation can be found [here](https://github.com/DataDog/system-tests/blob/main/docs/scenarios/onboarding.md).
```
